assets/javascripts/kitten/components/form/text-input/test.js

```diff
@@ -106,4 +96,14 @@ describe('<TextInput />', () => {
       expect(component).toMatchSnapshot()
     })
   })
+
+  describe('with `variant` prop', () => {
+    beforeEach(() => {
+      component = renderer.create(<TextInput variant="andromeda" />).toJSON()
```

Comment:
Peut-être que ça serait plus pertinent avec une variante différente ?

```suggestion
      component = renderer.create(<TextInput variant="orion" />).toJSON()
```